packages/db/postgres/migrations/016-add-uploads-user-id-index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS upload_user_id_deleted_at_idx ON upload (user_id) WHERE deleted_at IS NULL;
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS upload_user_id_deleted_at_idx ON upload (user_id) WHERE deleted_at IS NULL;
```

This needs to concurrently so that the operation does not lock the database